packages/forklift-console-plugin/src/modules/Providers/hooks/utils/constants.ts

```diff
@@ -1,15 +1,11 @@
-export const DEFAULT_FIELDS_TO_COMPARE = [
-  'vmCount',
-  'networkCount',
-  'storageClassCount',
-  'regionCount',
-  'projectCount',
-  'imageCount',
-  'volumeCount',
-  'volumeTypeCount',
-  'datacenterCount',
-  'clusterCount',
-  'hostCount',
-  'storageDomainCount',
-  'datastoreCount',
-];
+/**
+ * List of fields to ignore/skip comparing when checking if an inventory's object has changed and required
+ * re-rendering.
+ *
+ * Note: The fields included in this list have values which are changed occasionally in inventory
+ * while not displayed in UI and therefore if not ignored it will
+ * generate frequent unnecessary re-rendering.
+ *
+ * Need to add more fields to this if required.
+ */
+export const DEFAULT_FIELDS_TO_AVOID_COMPARING = ['revision', 'revisionValidated', 'storageUsed'];
```

Comment:
Note:
we use `useProviderInventory` for all the lists the use inventory, when avoiding coparing 'storageUsed' we need to see that we don't use this fields in all places we use this method.

Reply:
I checked both the plugin and backend code. 
We don't use `storageUsed` in our plugin code and it seems like being used in backend only for the VM model (vsphere and ova providers).
So I think it's safe to ignore that field.